### ISSUE TITLE
NPE opening new tab

### ISSUE THREAD

**Issue by syphyr** [bromite/bromite#1611]
### Bromite version

Version: `95.0.4638.79`
Arch:  `arm64`
Android version: `7.1`
Device model: `SM-T813`

### Flags

These are the flags changed under `chrome://flags`:

### Is this bug about the SystemWebView?

No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No

### Is this bug happening in an incognito tab?
Yes

### Allow ads for the site; is the bug caused by the adblocker?
No

### Is the bug a crash?

Yes, this is a copy/paste of the crash dump:
```
11-22 02:38:30.414 10051 10051 E AndroidRuntime: FATAL EXCEPTION: main
11-22 02:38:30.414 10051 10051 E AndroidRuntime: Process: org.bromite.bromite, PID: 10051
11-22 02:38:30.414 10051 10051 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke interface method 'int Ew1.n(org.chromium.chrome.browser.tab.Tab)' on a null object reference
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at rB.i(chromium-ChromePublic.apk-stable-1637054280:1)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at rB.j(chromium-ChromePublic.apk-stable-1637054280:4)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at rB.f(chromium-ChromePublic.apk-stable-1637054280:1)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at pu1.e(chromium-ChromePublic.apk-stable-1637054280:2)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at B10.onClick(chromium-ChromePublic.apk-stable-1637054280:7)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.view.View.performClick(View.java:5637)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:22433)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
11-22 02:38:30.414 10051 10051 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

### Describe the bug

Bromite crashes when closing all tabs and then opening a new tab on Android arm64

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Open Bromite in incognito
2. Close new tab (all tabs are closed)
3. Select "+" to open new tab

Do not write "any website", specify which URLs can be used to reproduce the issue.

### Expected behavior

Open new tab without NPE

**Comment by kokukyocho** [comment]
Hello @syphyr!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md); the issue will be automatically re-opened afterwards.

If this is not an issue nor a feature request and you want to start a discussion then you can use [GitHub Discussions](https://github.com/bromite/bromite/discussions).

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.

**Comment by uazo** [comment]
the deobfuscated stack trace:
```
11-22 02:38:30.414 10051 10051 E AndroidRuntime: FATAL EXCEPTION: main
11-22 02:38:30.414 10051 10051 E AndroidRuntime: Process: org.bromite.bromite, PID: 10051
11-22 02:38:30.414 10051 10051 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke interface method 'int org.chromium.chrome.browser.tabmodel.TabList.indexOf(org.chromium.chrome.browser.tab.Tab)' on a null object reference
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at org.chromium.chrome.browser.tabmodel.ChromeTabCreator.createNewTab(ChromeTabCreator.java:143)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at org.chromium.chrome.browser.tabmodel.ChromeTabCreator.launchUrl(ChromeTabCreator.java:332)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at org.chromium.chrome.browser.tabmodel.ChromeTabCreator.launchUrl(ChromeTabCreator.java:315)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at org.chromium.chrome.browser.tabmodel.TabCreator.launchNTP(TabCreator.java:97)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at org.chromium.chrome.browser.ui.tablet.emptybackground.EmptyBackgroundViewTablet$1.onClick(EmptyBackgroundViewTablet.java:63)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.view.View.performClick(View.java:5637)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.view.View$PerformClick.run(View.java:22433)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:751)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:95)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6186)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Method.java)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
11-22 02:38:30.414 10051 10051 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

**Comment by uazo** [comment]
I confirm the crash.

> Open Bromite in incognito

@syphyr do you have the always incognito active, right?

**Comment by syphyr** [comment]
> I confirm the crash.
> 
> > Open Bromite in incognito
> 
> @syphyr do you have the always incognito active, right?

Yes

**Comment by csagan5** [comment]
Fixed in `96.0.4664.54`.
